projects/admin/src/app/record/editor/remote-autocomplete/remote-autocomplete.component.ts

```diff
@@ -97,23 +97,51 @@ export class RemoteAutocompleteInputTypeComponent extends FieldType implements O
             return [];
           }
           results.hits.hits.map(hit => {
-            names.push({
-              name: this.getName(hit.metadata),
-              ref: `https://mef.rero.ch/api/mef/${hit.metadata.pid}`,
-              query,
-              category: this.translateService.instant('link to authority')
-            });
+            for (const source of ['idref', 'gnd']) {
+              if (hit.metadata[source]) {
+                names.push(this.getNameRef(hit.metadata, source, query));
+              }
+            }
           });
           return names;
         })
       );
   }
 
+  getNameRef(metadata, source, query) {
+    const upperSource = source.toUpperCase( );
+    return {
+      name: this.getNameSource(metadata[source]),
+      ref: `https://mef.rero.ch/api/${source}/${metadata[source].pid}`,
+      query,
+      category: this.translateService.instant(`link to authority ${upperSource}`)
```

Comment:
you can replace upperSource by source. This will be more clear for the translators.

Reply:
ok